scripts/k8s-local-cluster-test/delete-cluster.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
please name this script `scripts/delete-kind-cluster.sh` (and remove the directory "k8s-local-cluster-test")